config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  namespace :api do
+    namespace :v1 do
+      resources :departments, defaults: { format: :json }
```

Comment:
no need for mentioning `defaults: { format: :json }`

Reply:
without defaults: { format: :json } I get error:
ActionController::UnknownFormat in Api::V1::DepartmentsController#index
Api::V1::DepartmentsController#index is missing a template for this request format and variant.
request.formats: ["text/html"]
request.variant: []
why can this error occur?